app/assets/javascripts/views/footer/footer_view.ts

```diff
@@ -301,7 +301,7 @@ class FooterViewCtrl extends PureViewCtrl<
       CollectionSort.Title,
       'asc',
       (theme: SNTheme) => {
-        return theme.package_info && theme.package_info.dock_icon;
+        return theme.package_info && theme.package_info.dock_icon != null;
```

Comment:
```suggestion
        return theme.package_info && !isNullOrUndefined(theme.package_info.dock_icon);
```

Reply:
But, why a function for a language feature?

Reply:
What if it's equal to undefined here?

Reply:
The `x != null`  checks for both `null` and `undefined`.
If I had used `x !== null` then it would only check for `null`.

Reply:
Ah ok. So why check for null instead of undefined if both are the same?

Reply:
Just a habit.

Reply:
If both the same let's go with undefined, since that's far more prevalent in our codebases than null.